File: swift/scripts/HelloBranch.swift
import Foundation

let helloWorld = "Hello, world!"

if helloWorld == "Good morning" {
  print("morning")
} else if helloWorld.contains("Hello") {
  print("noon")
} else {
  print("night")
}

switch helloWorld {
  case "Hello, world!":
    print("noon")
  case "Good morning, world!":
    print("morning")
  default:
    print("night")
}

File: nodejs/scripts/helloBranch.js
const helloWorld = 'Hello, world!'

if (helloWorld === 'Good morning') {
  console.log('morning')
} else if (helloWorld.includes('Hello')) {
  console.log('noon')
} else {
  console.log('night')
}

switch (helloWorld) {
  case 'Hello, world!':
    console.log('noon')
    break
  case 'Good morning, world!':
    console.log('morning')
    break
  default:
    console.log('night')
}
